haystack/utils/app_loading.py

```diff
@@ -14,6 +15,14 @@ def haystack_get_app_modules():
     """Return the Python module for each installed app"""
     return [i.module for i in apps.get_app_configs()]
 
+def haystack_get_search_indexes():
+    """Return all modules to collect search indexes in"""
+    app_modules = [(app_mod, '%s.search_indexes' % app_mod.__name__) for app_mod in haystack_get_app_modules()]
+    custom_search_indexes = getattr(settings, 'HAYSTACK_SEARCH_INDEX_MODULES', [])
+    app_modules.extend([(None, search_index) for search_index in custom_search_indexes])
```

Comment:
Would it make more sense to set the app_mod here instead of setting to None? That way a user could set the `HAYSTACK_SEARCH_INDEX_MODULES` to a directory instead of listing out specific files. That would let the `module_has_submodule` check in loading.py handle collecting from multiple modules in the dir.